api/index.js

```
@@ -0,0 +1,57 @@
+import syncRouter from './sync.js';
+
+const router = express.Router();
+
+// ─── Health Check ─────────────────────────────────────────────────────────────
+router.get('/health', (req, res) => {
+  res.json({
+    status: 'ok',
+    version: '22.0.0',
+    timestamp: new Date().toISOString(),
+    services: {
+      stripe: !!process.env.STRIPE_SECRET_KEY,
```

ESLint

[error] 'process' is not defined.

(no-undef)